rotkehlchen/types.py

```diff
@@ -310,6 +310,17 @@ def identifier(self) -> str:
         return self.tx_hash + self.from_address.replace('0x', '') + str(self.nonce)
 
 
+class ChainID(SerializableEnumMixin):
```

Comment:
Perhaps add small docstring here explaining what the class is and represents and also as discussed reimplement serialize/deserialize from db just to be a bit more future-proof in the way we can change the code.

Add it in the docstrings of the function that they exist to make it easier to make changes to serialize/deserialize in the future if need be